Show error message on invalid credentials

vardhan-saniko opened the pull request:
<img width="660" alt="Screen Shot 2022-03-07 at 10 48 33 AM" src="https://user-images.githubusercontent.com/90535201/157089428-080bd246-4486-4cb5-8e84-19b3182b02a6.png">

tjquinn1 commented on src/components/auth/Signin.vue:
```diff
@@ -1,6 +1,11 @@
     v-card(class="pa-10 ma-auto text-center" width="300")
         h2(class="mb-8") Welcome.
+        <div v-if="errorMessage">
+            <v-alert color="red">
```
Change this to use the error theming as stated here. https://vuetifyjs.com/en/features/theme/#light-and-dark. This makes it easier to switch colors and not have to change it in a bunch of places

vardhan-saniko commented:
Addressed comment.
<img width="336" alt="Screen Shot 2022-03-11 at 8 22 12 AM" src="https://user-images.githubusercontent.com/90535201/157897440-86015ccd-0fd3-47f9-9850-691369e3fdd6.png">

tjquinn1 commented on src/components/auth/Signin.vue:
```diff
@@ -52,19 +59,21 @@ export default class Signin extends Vue {
+		let catchCallback = function() {
+			this.errorMessage = true;
+		};
+		catchCallback = catchCallback.bind(this);
```
This is needed. Just use '=>' which will bind 'this'. You catch should be 
```
.catch((error) => {
console.log(error);
})
```
